### ISSUE TITLE
[rocsparse] requires rocprim as dependency

### ISSUE THREAD

**Issue by cheesinglee** [rocm-arch/rocm-arch#175]
Trying to build 3.3.0-1, I get this configure error:

```==> Starting build()...
-- The CXX compiler identification is Clang 10.0.0
-- Check for working CXX compiler: /opt/rocm/hcc/bin/hcc
-- Check for working CXX compiler: /opt/rocm/hcc/bin/hcc - works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- HCC compiler set; ROCm backend selected [ CXX=/opt/rocm/hcc/bin/hcc cmake .. ]
-- Found Git: /usr/bin/git (found version "2.26.2") 
CMake Error at cmake/Dependencies.cmake:30 (find_package):
  By not providing "Findrocprim.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "rocprim", but
  CMake did not find one.

  Could not find a package configuration file provided by "rocprim" with any
  of the following names:

    rocprimConfig.cmake
    rocprim-config.cmake

  Add the installation prefix of "rocprim" to CMAKE_PREFIX_PATH or set
  "rocprim_DIR" to a directory containing one of the above files.  If
  "rocprim" provides a separate development package or SDK, be sure it has
  been installed.
Call Stack (most recent call first):
  CMakeLists.txt:85 (include)


-- Configuring incomplete, errors occurred!
```

If I build and install rocprim first, rocsparse builds and installs correctly. I don't really understand the rocm stack well, so I don't know if it is more appropriate to add rocprim to `depends` or `makedepends`. If someone tells me, I can submit a PR.

**Comment by tpkessler** [comment]
Hi @cheesinglee! The packages in `rocm-libs` are still  work in progress. I submitted PR #160 for `rocsparse` with which I'm able to build `rocsparse` in a clean chroot.

**Comment by tpkessler** [comment]
Solved with #160
